test/sanity/validate-modules/ignore.txt

```diff
@@ -520,6 +520,7 @@ lib/ansible/modules/cloud/vultr/vr_dns_record.py E326
 lib/ansible/modules/cloud/vultr/vr_firewall_group.py E324
 lib/ansible/modules/cloud/vultr/vr_firewall_rule.py E323
 lib/ansible/modules/cloud/vultr/vr_firewall_rule.py E324
+lib/ansible/modules/cloud/vultr/vr_region_facts.py E324
```

Comment:
New ignores should not be added. Instead, the underlying issue should be fixed.

If the issue is in existing common code (docs fragments or module_utils), then the fix should be in a separate PR.

Reply:
@mattclay agree. I did that while https://github.com/ansible/ansible/pull/42464 gets merged, so the CI feedback is green. Once merged, I'll update this PR removing this line from here.